FlowCrypt/Models/Realm Models/ContactKeyObject.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ContactKeyObject.swift
+//  FlowCrypt
+//
+//  Created by Roma Sosnovsky on 11/10/21
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+final class PubKeyObject: Object {
+    @Persisted(primaryKey: true) var key: String = ""
```

Comment:
The primary fingerprint (which every OpenPGP key has) should be the primary key on this table.

For column `key` let's call it `armored` instead. It means the key in armored format, with the `----BEGIN PGP PUBLIC KEY-----` header and so on, which is what we're storing.

Reply:
That means the `PublicKey` type could be updated too to reflect.